cranelift/codegen/src/isa/aarch64/lower/isle.rs

```diff
@@ -152,17 +152,14 @@ where
                         let imm =
                             MoveWideConst::maybe_with_shift(((!imm16) & 0xffff) as u16, i * 16)
                                 .unwrap();
-                        self.emitted_insts
```

Comment:
Can `emitted_insts` be made private after this? That would prevent similar issues in the future.

Reply:
Unfortunately, that doesn't work because different backends have slightly different definitions of `emit` so it is part of the ISLE-generated trait that is implemented for each backend and those implementations need to be able to push onto `emitted_insts`.